#  Vector of Polygons for a region
Header header
string label
geometry_msgs/Point32 normal
Polygon[] polygons
